src/backends/beamcoder.ts

```diff
@@ -518,7 +514,7 @@ export class BeamcoderExtractor extends BaseExtractor implements Extractor {
         return packet as Packet;
     }
 
-    _setFrameDataToImageData(frame: beamcoder.Frame, target: Uint8ClampedArray) {
+    _setFrameDataToRGBABufferTarget(frame: beamcoder.Frame, rgbaBufferTarget: Uint8ClampedArray) {
```

Comment:
/nit
consider making this a static method -- I don't see any use of `this.`

Reply:
Fixed